### DISCUSSION TITLE
Adding a CLK_OUT pin to a device tree?

### DISCUSSION THREAD

**Discussion by aquilliam** [zephyrproject-rtos/zephyr#62728] [Q&A]
Hello,
I have an STM32WB running Zephyr and I would like to provide a 40Mhz clock signal to a peripheral from the MCU. I can do this using STM's raw registers, but I have no idea how to do it in Zephyr, I haven't been able to find any examples of configuring a clock out pin.

**Comment by erwango** [comment]
This is actually a work on progress but functional.
See https://github.com/zephyrproject-rtos/zephyr/issues/31912
And you'd need something like: https://github.com/zephyrproject-rtos/zephyr/pull/60388/
